modules/prediction/scenario/right_of_way/right_of_way.cc

```diff
@@ -34,8 +37,120 @@ using hdmap::LaneInfo;
 using hdmap::OverlapInfo;
 using ConstLaneInfoPtr = std::shared_ptr<const LaneInfo>;
 
+std::unordered_set<std::string> RightOfWay::ego_back_lane_id_set_;
+std::unordered_set<std::string> RightOfWay::ego_front_lane_id_set_;
```

Comment:
Local variables should be `static` or in anonymous namespace to avoid pollute global namespace.